taxonomies/ingredients.txt

```diff
@@ -19271,7 +19287,7 @@ fr:tournesol oléique
 en:high oleic sunflower oil
 ca:oli refinat de gira-sol alt oleic
 de:High Oleic-Sonnenblumenöl
-es:aceite de girasol alto oleico,aceite refinado de girasol alto oleico,huile de tournesol haut oléique
+es:aceite de girasol alto oleico,aceite refinado de girasol alto oleico,huile de tournesol haut oléique,omega 9
```

Comment:
why "omega 9" ?
also "huile de tournesol haut oléique" should be in the fr: after (but it's from another change)

Reply:
Omega 9 from these 3:
https://es.openfoodfacts.org/ingrediente/omega-9

Reply:
"Aceite de glrasol con alto contenido de acido oleico (omega 9),"
--> we should not make "Omega 9" a synonym of "aceite de girasol alto oleico". There are lots of other possible sources of Omega 9.

Instead we can add Omega 9 as a single ingredient. (without linking it to the different oils).

Reply:
ok